pkg/controller/node/controller.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package node
+
+import (
+	"context"
+	"time"
+
+	gogotypes "github.com/gogo/protobuf/types"
+	"github.com/onosproject/onos-lib-go/pkg/env"
+
+	"github.com/onosproject/onos-lib-go/pkg/errors"
+
+	topoapi "github.com/onosproject/onos-api/go/onos/topo"
+
+	"github.com/onosproject/onos-config/pkg/controller/utils"
+	"github.com/onosproject/onos-config/pkg/store/topo"
+	"github.com/onosproject/onos-lib-go/pkg/controller"
+	"github.com/onosproject/onos-lib-go/pkg/logging"
+)
+
+const (
+	defaultTimeout            = 30 * time.Second
+	defaultExpirationDuration = 30 * time.Second
+	defaultGRPCPort           = 5150
+	defaultGNMIPort           = 9339
+)
+
+var log = logging.GetLogger("controller", "node")
```

Comment:
not sure yet about the name of the controller @kuujo

Reply:
I think this is a fine name